src/wizards/modify_task.xml

```diff
@@ -164,6 +179,21 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
                   <xsl:value-of select="translate(substring (date:date-time (), 1, 19), '-:T', '')"/>
                 </name>
                 <comment>Automatically generated by wizard</comment>
+                <icalendar>
+                  <xsl:text>PRODID:-//Greenbone.net//NONSGML Greenbone Security Manager</xsl:text>
+                  <xsl:call-template name="newline"/>
+                  <xsl:text>VERSION:2.0</xsl:text>
+                  <xsl:call-template name="newline"/>
+                  <xsl:text>BEGIN:VEVENT</xsl:text>
+                  <xsl:call-template name="newline"/>
+                  <xsl:text>DTSTART:</xsl:text>
+                  <xsl:value-of select="$start"/>
+                  <xsl:call-template name="newline"/>
+                  <xsl:text>END:VEVENT</xsl:text>
+                  <xsl:call-template name="newline"/>
+                  <xsl:text>END:VCALENDAR</xsl:text>
+                  <xsl:call-template name="newline"/>
+                </icalendar>
                 <first_time>
```

Comment:
It looks like the old `<first_time>` element could be removed here as well.

Reply:
Done in 83660e5.